spec/factories/rents.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :rent do
+    user { Faker::Name.first_name }
+    book { Faker::Book.title }
+    rent_start { Faker::Date.today }
+    rent_end { Faker::Date.today + 5 }
```

Comment:
Esto es una duda, no una coreccion, pero, los campos se llaman asi en apiary ?

Reply:
Jajajajajaj esto es un poco confuso
![image](https://user-images.githubusercontent.com/11413458/55155432-18f05c00-5136-11e9-8161-de9ac3f643eb.png)
La card de Trello dice que no uses nombres como from y to, pero en apiary se usan esos nombres. Tomo nota para corregirlo en futuros trainnings :wink:

Reply:
Volviendo a la corrección: coincido! y siempre me parece redundante poner antes de la variable el nombre del modelo. Ejemplo: Si usaramos `rent_name` me parecería redundante el `rent_` del principio, en ese caso sólo lo llamaria `name`.
Para este caso en particular, yo llamaría a estas variables `start_date` y `end_date` :man_shrugging: pero me da igual